tls_icon.php

```diff
@@ -5,6 +5,7 @@ class tls_icon extends rcube_plugin
 	const POSTFIX_TLS_REGEX = "/\(using (TLS.*)\) \(/im";
 	const POSTFIX_LOCAL_REGEX = "/\([a-zA-Z]*, from userid [0-9]*\)/im";
 	const SENDMAIL_TLS_REGEX = "/\(version=(TLS.*)\)(\s+for|;)/im";
+    const STALWART_TLS_REGEX = "/\(using (TLSv1\.[23] with cipher .+)\) by .+ \(Stalwart SMTP\)/im";
```

Comment:
This regex appears similar to the Postfix regex. Is there a specific reason why the existing Postfix regex doesn't work for Stalwart?

Reply:
Also, I don't like having overly specific regexes. Currently TLSv1.2 and TLSv1.3 are the only protocol versions commonly used, but to anticipate a future TLSv1.3 or TLSv2.0 I don't think the regex needs to match only  `TLSv1\.[23]`.